apps/retrospective/serializers/sprint_task.go

```diff
@@ -4,30 +4,31 @@ import "time"
 
 // SprintTask ...
 type SprintTask struct {
-	ID                   uint
-	Key                  string
-	URL                  string
-	Summary              string
-	Description          string
-	Type                 string
-	Status               string
-	Priority             string
-	IsTrackerTask        bool
-	IsInvalid            bool
-	Rating               int8
-	Estimate             float64
-	TotalPointsEarned    float64
-	PointsEarned         float64
-	Assignee             string // Assignee of the task
-	Owner                string // Owner of the task across the sprints
-	TaskParticipants     string // Participants of the task across the sprints
-	SprintOwner          string // Owner of the task in the sprint
-	SprintParticipants   string // Participants of the task in the current sprint
-	SprintOwnerTime      uint   // Time spent on the task by the Sprint Owner in the sprint
-	SprintOwnerTotalTime uint   // Total time spent on the task by the Sprint Owner across the sprints
-	SprintTime           uint   // Time spent on the task in the sprint
-	TotalTime            uint   // Total time spent on the task across the sprints
-	DoneAt               *time.Time
+	ID                     uint
+	Key                    string
+	URL                    string
+	Summary                string
+	Description            string
+	Type                   string
+	Status                 string
+	Priority               string
+	IsTrackerTask          bool
+	IsInvalid              bool
+	Rating                 int8
+	Estimate               float64
+	TotalPointsEarned      float64
+	PointsEarned           float64
+	Assignee               string // Assignee of the task
+	Owner                  string // Owner of the task across the sprints
+	TaskParticipants       string // Participants of the task across the sprints
+	SprintOwner            string // Owner of the task in the sprint
+	SprintParticipants     string // Participants of the task in the current sprint
+	SprintActiveMemberTime uint   // Time spent on the task by the Current Login Member in the sprint
```

Comment:
```suggestion
	SprintCurrentMemberTime uint   // Time spent on the task by the Current Login Member in the sprint
```
or something more relevant.

Reply:
changed